mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -1172,6 +1172,7 @@ func expandKubeletConfig(v interface{}) *container.NodeKubeletConfig {
 	kConfig := &container.NodeKubeletConfig{}
 	if cpuManagerPolicy, ok := cfg["cpu_manager_policy"]; ok {
 		kConfig.CpuManagerPolicy = cpuManagerPolicy.(string)
+		kConfig.ForceSendFields = append(kConfig.ForceSendFields, "CpuManagerPolicy")
```

Comment:
This will cause the field to be sent even if unset - per https://github.com/hashicorp/terraform-provider-google/issues/15767 that might not be necessary / desired?

Reply:
It's been a few, so I may have to run through some different scenarios to remember exactly why this ended up seemingly being needed.

IIRC, there was some kind of issue when I didn't have it set -- at the least, I did try it first without that line added.

If you're already setup to test it, you could try building the provider with this line removed and step through some of the tests that I did. If I've got time to dig into it, I will try to see as well.

Reply:
I did at least a basic test with this line removed again, and a config with other `kubelet_config` stuff set seemed to do what I'd expect.

Another scenario to look at is when the nodepool is managed separately (where nodepool updates _do_ work)... I think there's a case where removing the attribute when it was set might cause an issue when the attribute doesn't get set in the API call. That said, I haven't been able to induce that issue thus far with this example, and `TestAccContainerNodePool_withKubeletConfig` passes, so I'm pushing up an update that removes this again. I didn't test `""` but since that's not a documented value, probably doesn't matter too much even if that did break somehow.

```hcl
resource "google_container_cluster" "test_cfs_quota" {
  name               = "test-cfs-quota"
  location           = "us-central1-f"
  initial_node_count = 1

  node_config {
    kubelet_config {
      cpu_cfs_quota                          = true
      insecure_kubelet_readonly_port_enabled = "FALSE"
    }
  }

  deletion_protection = false
  network             = "default"
  subnetwork          = "default"
}

resource "google_container_node_pool" "test" {
  name       = "secondary-test"
  cluster    = google_container_cluster.test_cfs_quota.id
  node_count = 1

  node_config {
    kubelet_config {
      cpu_cfs_quota                          = true
      cpu_manager_policy                     = "none"
      insecure_kubelet_readonly_port_enabled = "FALSE"
    }
  }
}
```